components/apply-link/index.js

```diff
@@ -22,6 +22,21 @@ const StyledLink = styled.a`
   text-decoration-line: none;
   font-family: inherit;
 
+  :hover {
+    background: linear-gradient(
+      90.03deg,
+      rgba(255, 104, 186, 0.25) 0.39%,
+      rgba(237, 194, 129, 0.25) 99.97%
+    );
+    color: white;
+    span {
+      display: none;
```

Comment:
I'm not 100% sure if accessibility tools handle hover, but if they do this `display: none` will remove the button text from the tree and the `:before` content might not be picked up in its place.
I was trying to think of a good way that the text could be swapped out and be accessible, but all I could think of is js event handlers which isn't as clean as this.

Reply:
I see what you mean. Yeah, I am not sure what would be best here. Any suggestions on how to test it?

Reply:
I think testing it out in [ChromeVox](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) is probably a decent test.